cpu/cc26xx_cc13xx/include/cc26xx_cc13xx_rfc_common_cmd.h

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright (C) 2020 Locha Inc
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup         cpu_cc26xx_cc13xx
+ * @{
+ *
+ * @file
+ * @brief           CC26xx/CC13xx RF Core common commands definitions
+ *
+ * @author          Jean Pierre Dudey <jeandudey@hotmail.com>
+ */
+
+#ifndef CC26XX_CC13XX_RFC_COMMON_CMD_H
+#define CC26XX_CC13XX_RFC_COMMON_CMD_H
+
+#include <stdint.h>
+
+#include "cc26xx_cc13xx_rfc_mailbox.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Direct commands
+ * @{
+ */
+#define RFC_CMD_STOP (0x0402) /**< Stop a running command */
+/** @} */
+
+/**
+ * @brief   Radio operation start trigger
+ */
+typedef struct {
+    uint8_t type:4; /**< The type of trigger */
+    uint8_t ena_cmd:1; /**< 0h = No alternative trigger command,
+                            1h = CMD_TRIGGER can be used as an alternative
+                            trigger */
+    uint8_t trigger_no:2; /**< The trigger number of the CMD_TRIGGER command that
+                               triggers this action */
+    uint8_t past_trig:1; /**< 0: A trigger in the past is never triggered, or for
+                              start of commands, give an error */
+} rfc_trigger_t;
+
+/**
+ * @brief   Condition for running next operation
+ */
+typedef struct {
+    uint8_t rule:4; /**< Rule for how to proceed */
+    uint8_t skip_no:4; /**< Skip number + 1 if the rule involves skipping */
+} rfc_cond_t;
+
+/**
+ * @brief   General radio operation.
+ */
+typedef struct {
+    uint16_t command_no; /**< The command ID number */
+    uint16_t status; /**< An integer telling the status of the command. */
+    void *next_op; /**< Pointer to the next operation to run */
+    rfc_ratmr_t start_time; /**< Absolute or relative start time */
+    rfc_trigger_t start_trigger; /**< Identification of the trigger that
+                                         starts the operation */
+    rfc_cond_t condition; /**< Condition for running next command */
+} __attribute__ ((aligned (4))) rfc_op_t;
```

Comment:
4 should be the default alignment anyway

Reply:
Fixed, thanks for pointing it!